src/platform/plugins/shared/esql/server/services/esql_service.ts

```diff
@@ -65,6 +69,11 @@ export class EsqlService {
       flat_settings: true,
     })) as IndexModeResponse;
 
+    const ccsSources = (await client.indices.resolveIndex({
```

Comment:
Since `*:*` is expensive, we may want to check if the `FROM` pattern has any remote clusters and only do this if it does?

Reply:
Yes I did that, it runs only if the users have a remote cluster in the from command